pyquil/operator_estimation.py

```diff
@@ -214,71 +209,6 @@ def measure_observables(
     symmetrization = tomo_experiment.symmetrization
     reset = tomo_experiment.reset
 
-    if n_shots is not None:
-        warnings.warn(
-            "'n_shots' has been deprecated; if you want to set the number of shots "
-            "for this run of measure_observables please provide the number to "
-            "Program.wrap_in_numshots_loop() for the Quil program that you provide "
-            "when creating your TomographyExperiment object. For now, this value will "
-            "override that in the TomographyExperiment, but eventually this keyword "
-            "argument will be removed.",
-            FutureWarning,
-        )
-        shots = n_shots
-    else:
-        if shots == 1:
-            warnings.warn(
-                "'n_shots' has been deprecated; if you want to set the number of shots "
-                "for this run of measure_observables please provide the number to "
-                "Program.wrap_in_numshots_loop() for the Quil program that you provide "
-                "when creating your TomographyExperiment object. It looks like your "
-                "TomographyExperiment object has shots = 1, so for now we will change "
-                "that to 10000, which was the previous default value.",
-                FutureWarning,
-            )
-            shots = 10000
```

Comment:
You'll notice we call `Program.wrap_in_numshots_loop(10000)` in a few tests, as the new way to do this.